packages/@aws-cdk/aws-ecs/test/fargate/fargate-task-definition.test.ts

```diff
@@ -241,8 +242,106 @@ describe('fargate task definition', () => {
         taskDefinition.taskRole;
       }).toThrow('This operation requires the taskRole in ImportedTaskDefinition to be defined. ' +
         'Add the \'taskRole\' in ImportedTaskDefinitionProps to instantiate ImportedTaskDefinition');
+    });
+
 
+    test('runtime testing for windows container', () => {
+      // GIVEN
+      const stack = new cdk.Stack();
+      new ecs.FargateTaskDefinition(stack, 'FargateTaskDef', {
+        cpu: 1024,
+        memoryLimitMiB: 2048,
+        runtimePlatform: {
+          operatingSystemFamily: OperatingSystemFamily.WINDOWS_SERVER_2019_CORE,
+          cpuArchitecture: CpuArchitecture.X86_64,
+        },
+      });
 
+      // THEN
+      expect(stack).toHaveResourceLike('AWS::ECS::TaskDefinition', {
+        Cpu: '1024',
+        Family: 'FargateTaskDef',
+        Memory: '2048',
+        NetworkMode: 'awsvpc',
+        RequiresCompatibilities: [
+          ecs.LaunchType.FARGATE,
+        ],
+        RuntimePlatform: {
+          CpuArchitecture: 'X86_64',
+          OperatingSystemFamily: 'WINDOWS_SERVER_2019_CORE',
+        },
+        TaskRoleArn: {
+          'Fn::GetAtt': [
+            'FargateTaskDefTaskRole0B257552',
+            'Arn',
+          ],
+        },
+      });
     });
+
+    test('runtime testing for windows container', () => {
```

Comment:
This test has the exact same name as the previous one. Should this one be called 'runtime testing for linux container'?